src/base/bittorrent/session.cpp

```diff
@@ -270,6 +270,7 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("utpMixedMode"), m_isUTPEnabled)
```

Comment:
>Also the name should be uTPMixedMode

OK

>So I would say the default here should be false.

Do you think we should also turn uTP off by default? otherwise, with uTP on + prefer_tcp, uTP would be useless in theory.